terraform/main.tf

```diff
@@ -147,38 +153,73 @@ resource "azurerm_virtual_machine_scale_set" "scaleset" {
     type_handler_version    = "2.0"
     settings                = <<SETTINGS
         {
-         "commandToExecute": "cmd",
+          "commandToExecute": "${var.command}",
           "fileUris": [
-            "https://<storagename>.blob.core.windows.net/<file>",
+            "${var.file1}",
+            "${var.file2}"
           ]
         }
       SETTINGS
   }
 }
 
-
-resource "azurerm_key_vault" "kv" {
+# IMPLEMENTATION NOTE: The Terraform AzureRM provider does not currently support access policies

```

Comment:
I'm not sure we really need this workaround here. Would be great to update when Terraform adds support for this scenario.